cmdx.py

```diff
@@ -2291,18 +2317,24 @@ def flatten(self, type=None):
         """Return members, converting nested object sets into its members
 
         Example:
-            >>> from maya import cmds
-            >>> _ = cmds.file(new=True, force=True)
-            >>> a = cmds.createNode("transform", name="a")
-            >>> b = cmds.createNode("transform", name="b")
-            >>> c = cmds.createNode("transform", name="c")
-            >>> cmds.select(a)
-            >>> gc = cmds.sets([a], name="grandchild")
-            >>> cc = cmds.sets([gc, b], name="child")
-            >>> parent = cmds.sets([cc, c], name="parent")
-            >>> mainset = encode(parent)
-            >>> sorted(mainset.flatten(), key=lambda n: n.name())
-            [|a, |b, |c]
+          >>> from maya import cmds
+          >>> _ = cmds.file(new=True, force=True)
+          >>> a = cmds.createNode("transform", name="a")
+          >>> b = cmds.createNode("transform", name="b")
+          >>> c = cmds.createNode("transform", name="c")
+          >>> cmds.select(a)
+          >>> gc = cmds.sets([a], name="grandchild")
+          >>> cc = cmds.sets([gc, b], name="child")
+          >>> parent = cmds.sets([cc, c], name="parent")
+          >>> mainset = encode(parent)
+          >>> sorted(mainset.flatten(), key=lambda n: n.name())
+          [<cmdx.DagNode : '|a'>, <cmdx.DagNode : '|b'>, <cmdx.DagNode : '|c'>]
```

Comment:
Un-indented these by 2 spaces so line 2331 isn't too long.

Reply:
There we go, now I can actually see what these look like. What do you think about:

```py
# Before
<cmdx.DagNode : '|node'>

# After
<cmdx.DagNode("|node")>
```

That way, it'll be more familiar and copy-pasteable as real code too.